ANSWERS.md

```diff
@@ -0,0 +1,25 @@
+1.  What are PropTypes used for?
+	specifying runtime assertions of what 'type' of data
+	a React component needs in order to render.
+
+2.  What is a lifecycle event in React?
+	What the 'lifecycle' refers to is how applications 'grow'	
+	or 'build', how data interacts with them being the 'life'
+	and finally when you either close the applicaiton or you 
+	switch tasks the app knows to stop putting computer 
+	resources into the closed or minimized operations.
+
```

Comment:
Hmmmm I think that's from the app level. In React component level, the lifecycle events are Mounting, Updating, and Unmounting phases of a react component